src/XCMRelayer.sol

```diff
@@ -209,4 +254,107 @@ contract AxelarXCMRelayer is Auth {
     function _parachainId() internal pure returns (bytes memory) {
         return abi.encodePacked(uint8(0), CENTRIFUGE_PARACHAIN_ID);
     }
+
+    // --- Utilities ---
+    function _centrifugeCall(bytes memory message) internal view returns (bytes memory) {
+        return abi.encodePacked(
+        // The Centrifuge liquidity-pools pallet index
+        centrifugeChainLiquidityPoolsGatewayPalletIndex,
+        // The `handle` call index within the liquidity-pools pallet
+        centrifugeChainLiquidityPoolsGatewayPalletProcessMsgCallIndex,
+        // We need to specify the length of the message in the scale-encoding format
+        messageLengthScaleEncoded(message),
+        // The connector message itself
+        message
```

Comment:
This must include the `sourceChain` and `sourceAddress` prefixed with the given length before any message encoding.

&rarr; `PalletIndex` + `CallIndex` + `LengthSourceChain` + `sourceChain` + `LengthSourceAddress` + `sourceAddress` + `MsgLength` + `Msg`